### ISSUE TITLE
Button tap doesn't bounce back its state

### ISSUE THREAD

**Issue by satvikmaker** [binggg/react-native-material-design-components#18]
Let me explain this behaviour. This is in reference to #9 and #10. But, clearly this issue is with the button component.
When I click on the button, it doesn't bounce back to released state. It is stuck in pressed state.

Phone - Samsung Galaxy S4
OS version - 5.0.1 and 5.1.1 (Tested on both)

I will look into this as well, will pull a PR soon. Just wrote it here so as to keep track of things.

**Comment by Ehesp** [comment]
Are you sure it's the button? If you go to the demo, with checkbox and hold down on the "box" item, the background effect is there... so it looks like it's more the ripple?

![image](https://cloud.githubusercontent.com/assets/842078/11608513/9f554102-9b63-11e5-8eb7-30a57ecc6449.png)

**Comment by satvikmaker** [comment]
@Ehesp Yes, I am talking about the button. The same issue is there on buttons as well.

**Comment by Ehesp** [comment]
I think I found out why this happens, if you hold down the item, then scroll your view, let go the "touch up" trigger seems like it's not being triggered.

**Comment by Ehesp** [comment]
Hmm saying that, it doesn't happen on the latest demo APK for me... Unless something has changed?
